lib/kiev/shoryuken/tree_path_tagger.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Kiev
+  module Shoryuken
+    class TreePathTagger
```

Comment:
Might be good idea to use it in rack middleware too.

Reply:
Um, no. The README explains why.
Long story short, it's only needed where "fanout" is not under Kiev's control and individual `tree_path`s cannot be assigned via `SubrequestHelper`. I don't see how this could happen with HTTP requests, but it can happen in any MQ, such as SNS+SQS or RabbitMQ's topics+queues.